lua/telescope/mappings.lua

```diff
@@ -26,6 +26,7 @@ mappings.default_mappings = config.values.default_mappings or {
 
       ["<Tab>"] = actions.toggle_selection + actions.move_selection_worse,
       ["<S-Tab>"] = actions.toggle_selection + actions.move_selection_better,
+      ["<M-Tab>"] = actions.toggle_all,
```

Comment:
I am currently at this point where i don't want to map new things on default. (Personally i don't have any vim mappings on MOD1)  and thats the problem with doing that. (I know tj mapped `<M-q>` but i also told him that this doesn't work for me and probably not for other people) yeah.

Leave it here for now. I'll think about it